published/005-2016-Feb-1.md

```diff
@@ -0,0 +1,209 @@
+    over IFPS will work in all future releases.
+* [**fs-repo-migrations**](//github.com/ipfs/fs-repo-migrations)
+  - [@chriscool](//github.com/chriscool) Improved fs-repo-migrations tests by checking that the objects are the same and checking that garbage collection does not change anything. He also cleaned up sharness tests.
+* [**http-api-spec**](//github.com/ipfs/http-api-spec)
+  - @RichardLitt finished logging all existing `ipfs` commands for the HTTP API Spec. This means that if you have any questions about how the HTTP API should work, you can see them either in master or as open PRs to that issue. One step closer to a fully functional API...
+* [**community**](//github.com/ipfs/community)
+  - @RichardLitt proposed using the new [GitHub Templates]() for IPFS. What do you think about this idea? Chime in [here](https://github.com/ipfs/community/issues/108).
+  - We now have [a list of community events](https://github.com/ipfs/community#events-calendar) for IPFS on the ipfs/community README. Do you have any that you'd like to add? Check out past events, too.
+* [**ipfs-hyperlog**](//github.com/noffle/ipfs-hyperlog) - @noffle launched am IPFS-compatible Merkle DAG that replicates based on scuttlebutt logs and causal linking. `ipfs-hyperlog` is a drop-in replacement for [@mafintosh](https://github.com/mafintosh)'s [`hyperlog`](https://github.com/mafintosh/hyperlog). Its key difference is that it creates a Merkle DAG that is *binary compatible* with IPFS objects. This means any node of any DAG built using ipfs-hyperlog can be replicated to and from the IPFS network as well!
+
+## Work in Progress
+
+* **Logo**: [@Kubuxu](//github.com/Kubuxu) worked on a new [IPFS logo](https://ipfs.io/ipfs/QmTgtbb4LckHaXh1YhpNcBu48cFY8zgT1Lh49q7q7ksf3M/). Check it out.
+
+## Community
+
+* **Conferences**: If you have any suggestions, there is a [new issue](https://github.com/ipfs/community/issues/105) where you can submit your proposals. The IPFS community will see if we can get to that conference and have a presence there.
+
+* [**Marmot Image Checker**](https://github.com/eris-ltd/marmot): This takes an image, asks the Google Cloud Vision API for three descriptions of the image, and compares those descriptions to a chosen list of words. If there is match, the image is added to the [toadserver](https://github.com/eris-ltd/toadserver), a smart contract and IPFS-Backed download server. [From Twitter](https://twitter.com/asciinema/status/701730719589126146)
+* [**asciienema**](https://github.com/asciinema/asciinema/releases/tag/v1.2.0) now allows playback on IPFS. [From Twitter](https://twitter.com/asciinema/status/701730719589126146)
+* [Introduction to IPFS](https://medium.com/@ConsenSys/an-introduction-to-ipfs-9bba4860abd0#.48ki3fymw): [@ChrisLundkvist](https://twitter.com/ChrisLundkvist) and [@ConsenSysAndrew](https://twitter.com/ConsenSysAndrew) wrote a fantastic write up about IPFS, from a less technical perspective.
```

Comment:
the write up is very technical, it's only the beginning that isn't. write a bit more about this, this was a big deal post that our friends wrote on our behalf. something like:

```
- Our friends and users at ConsenSys wrote an excellent "Introduction to IPFS", which starts with a less-technical preface, and then dives deep into a full explanation of how the IPFS object model works. It walks through multiple examples, including directory structures, version control systems, and blockchains. This is a great post to familiarize yourself with how the low level IPFS objects work, with graph visualizations and in-depth explanations. It also features a fantastic cover image! Thanks @ChrisLundkvist and @ConsenSysAndrew!

![](https://cdn-images-1.medium.com/max/2000/1*czZJ7mvEAqL4wNAg-jt9Ow.jpeg)
```